packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockPrivateChat.st

```diff
@@ -1,9 +1,10 @@
 chat
 mockPrivateChat
 
-	^ TCCChat new
+	^ TCCPrivateChat new
 		id: 1;
 		title: 'PrivateMockChat';
 		type: 'chatTypePrivate';
 		lastMessage: 'lastPrivateMessage';
-		canSendMessages: true
\ No newline at end of file
+		canSendMessages: true;
+		infoLoaded: true
```

Comment:
Should we add it if it makes no difference? What would yourself do in this case, I am not sure I fully understand what it does.

Reply:
You are right that it makes no difference. It would make a difference if the last method send (`infoLoaded: true` in this case) would return a value that is not the object itself (imagine it would return the new value of `infoLoaded`). Then this value would be returned by the whole method instead of the Chat. This is what yourself prevents. I'm not 100% sure about the conventions, whether they say to always add a yourself at the end of cascades, or not. So I'll leave that up to you, just wanted to mention it :)

Reply:
I changed it